developer-extension/src/background/utils.ts

```diff
@@ -1,4 +1,4 @@
-export function evaluateCodeInActiveTab(code: (arg?: string) => void, arg?: string) {
+export function evaluateCodeInActiveTab(code: (arg?: string) => void, arg?: any) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I don't see why you changed the type here, could you revert?

(Same for the function below)